### PR TITLE
Add unit metadata

### PR DESCRIPTION
**Pull request by belkassaby**
*Add unit metadata
*Add unit-api dependency to pom and tpd files

### CODE REVIEW

File: releng/org.eclipse.january.releng.target/org.eclipse.january.releng.target.tpd [hunk 1]
```diff
@@ -27,6 +27,7 @@ location "http://download.eclipse.org/tools/orbit/downloads/drops/S2016091616200
 	org.antlr.runtime [4.3.0,4.3.1) // CQ 11787
 	com.google.inject [3.0.0,3.0.1) // CQ 11788
 	org.apache.log4j [1.2.15,1.2.16) // CQ 11789
+	javax.measure.unit-api [1.0.0] // CQ 13959
```
Comment (jonahkichwacoders) [inline releng/org.eclipse.january.releng.target/org.eclipse.january.releng.target.tpd]: The CQ is done and the package will soon be in Orbit.

The location above needs to be updated to the appropriate repo once there is one built. Initially we can use an I build, but should change to S and then R build as they become available.

Comment (jonahkichwacoders) [inline releng/org.eclipse.january.releng.target/org.eclipse.january.releng.target.tpd]: Its there now in an M build (is that better or worse than an S build?) Doesn't matter, we should have an R build in a few weeks with unit-api to upgrade to:

http://download.eclipse.org/tools/orbit/downloads/
specifically:
http://download.eclipse.org/tools/orbit/downloads/drops/M20170818183741/

Thanks to Erwin for getting this into Orbit.

Comment (belkassaby) [reply releng/org.eclipse.january.releng.target/org.eclipse.january.releng.target.tpd]: I created a new pull-request : https://github.com/eclipse/january/pull/255 with the correct signed-off-by section in the commit message so the IP-validation passes.

**Comment by PeterC-DLS** [comment]
Closing for #256

**Pull request closed without merging**